cloudflare/resource_cloudflare_access_keys_configuration.go

```diff
@@ -0,0 +1,94 @@
+package cloudflare
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	cloudflare "github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceCloudflareAccessKeysConfiguration() *schema.Resource {
+	return &schema.Resource{
+		Read:   resourceCloudflareAccessKeysConfigurationRead,
+		Create: resourceCloudflareAccessKeysConfigurationCreate,
+		Update: resourceCloudflareAccessKeysConfigurationUpdate,
+		Delete: resourceCloudflareKeysConfigDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceCloudflareAccessMutualTLSCertificateImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"key_rotation_interval_days": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Computed: true,
+			},
+			"last_key_rotation_at": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"days_until_next_rotation": {
```

Comment:
This is going to need a custom [`DiffSuppressFunc`](https://www.terraform.io/docs/extend/schemas/schema-behaviors.html#diffsuppressfunc) as otherwise applying this on multiple days will trigger a `Apply`/`Plan` difference. Otherwise we might need to think about the intended behaviour here.